uint16                          tid
yesense_imu/YesenseImuLocation  location
yesense_imu/YesenseImuUtcTime   utc_time
yesense_imu/YesenseImuStatus    status